client/app/visualizations/chart/chart-editor.html

```diff
@@ -62,7 +62,7 @@
 
     </div>
 
-    <div class="form-group" ng-if="options.globalSeriesType != 'custom'">
+    <div class="form-group" ng-if="options.globalSeriesType != 'custom' && options.globalSeriesType != 'heatmap'">
```

Comment:
Replace it with `"['custom', 'heatmap'].indexOf(options.globalSeriesType) >= 0"` - it's easier to read and modify in future